client/my-sites/people/invite-people/index.jsx

```diff
@@ -110,16 +102,15 @@ class InvitePeople extends React.Component {
 		};
 	};
 
-	refreshFormState = () => {
-		const sendInvitesSuccess = InvitesSentStore.getSuccess( this.state.formId );
-
-		if ( sendInvitesSuccess ) {
+	refreshFormState = ( errors = {}, success = [] ) => {
+		if ( success.length && ! Object.keys( errors ).length ) {
 			this.setState( this.resetState() );
 			this.props.recordTracksEventAction( 'calypso_invite_people_form_refresh_initial' );
 			debug( 'Submit successful. Resetting form.' );
-		} else {
-			const sendInvitesErrored = InvitesSentStore.getErrors( this.state.formId );
-			const errors = get( sendInvitesErrored, 'errors', {} );
+			return;
+		}
+
+		if ( Object.keys( errors ).length ) {
```

Comment:
This is the second time we're doing a `Object.keys( errors )` in this method. Should we move it to a `const`?